misaligned.py

```diff
@@ -1,13 +1,19 @@
+formatted_string = []
 
 def print_color_map():
     major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
     minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
     for i, major in enumerate(major_colors):
         for j, minor in enumerate(minor_colors):
-            print(f'{i * 5 + j} | {major} | {minor}')
+            formatted_string = format_string(i, j, major, minor)
     return len(major_colors) * len(minor_colors)
 
+def format_string(i, j, major, minor):
+    formatted_string.append(f'{i*5 + j} | {major} | {minor}')
+    return formatted_string
 
 result = print_color_map()
 assert(result == 25)
+assert(formatted_string[0].find("|") == formatted_string[24].find("|"))

```

Comment:
Test case is still weak. How do we verify the content of color map effectively?
The number 5 used in the pair number calculation would lead to confusion or error in future when someone else is maintaining the code.
Also, will this code work if major color or minor color length is other than 5. Please think over it and write test cases accordingly to cover such cases.